### PR TITLE
[CLT-2020] Initial PR for Charlotte

### PR DESCRIPTION
**Pull request by alfonso-cabrera**

### CODE REVIEW

File: data/events/2020-charlotte.yml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  - name: "Ryan Kirk"
+  - name: "Phillip Shipley"
+  - name: "Meka Wakahulu"
+  - name: "James Strong"
+  - name: Brock Woodring
+    employer: Tire Intelligence
+  - name: "Quitra Radney"
+
+
+organizer_email: "organizers-charlotte@devopsdays.org" # Put your organizer email address here
```
Comment (bridgetkromhout) [inline data/events/2020-charlotte.yml]: This is actually just `charlotte@devopsdays.org` now. And you don't need a separate proposals email!

**Comment by mattstratton** [comment]
The build is failing due to this:
```
10:50:33 AM: Error: Error building site: "/opt/build/repo/content/events/2020-charlotte/propose.md:6:8": failed to render shortcode "cfp_dates": failed to process shortcode: "/opt/build/repo/themes/devopsdays-theme/layouts/shortcodes/cfp_dates.html:7:33": execute of template failed: template: shortcodes/cfp_dates.html:7:33: executing "shortcodes/cfp_dates.html" at <dateFormat "Monday, Jan 2, 2006" $cfp_date_start>: error calling dateFormat: unable to parse date: 2019-9-15T00:00:00-05:00
```

File: data/events/2020-charlotte.yml (hunk 1 above)
Comment (alfonso-cabrera) [reply data/events/2020-charlotte.yml]: Thanks, that is updated now!

**Comment by mattstratton** [comment]
Small nit - do you want to disable the "registration" and "program" links since they don't have info yet and add them later? Totally your call.

**Comment by alfonso-cabrera** [comment]
Hi Matt, good idea! I commented out those links from welcome.md and pushed the changes.

**Comment by mattstratton** [comment]
I commented them out of the data file too, so the nav items won't show up :)

**Comment by mattstratton** [comment]
@alfonso-cabrera 	

> I commented out those links from welcome.md and pushed the changes

You sure? I don't see this commit :)

**Pull request merged**